docs/configure.md

```diff
@@ -158,8 +199,7 @@ If you intend to make your Sourcegraph instance accessible on the Internet or an
 
    ```bash
    # This script requires https://github.com/sourcegraph/jy and https://github.com/sourcegraph/yj
-   FE=base/frontend/sourcegraph-frontend.Deployment.yaml
-   cat $FE | yj | jq '(.spec.template.spec.containers[] | select(.name == "frontend") | .env) += [{name: "TLS_CERT", valueFrom: {secretKeyRef: {key: "cert", name: "tls"}}}, {name: "TLS_KEY", valueFrom: {secretKeyRef: {key: "key", name: "tls"}}}]' | jy -o $FE
+   cat $(base/frontend/sourcegraph-frontend.Deployment.yaml) | yj | jq '(.spec.template.spec.containers[] | select(.name == "frontend") | .env) += [{name: "TLS_CERT", valueFrom: {secretKeyRef: {key: "cert", name: "tls"}}}, {name: "TLS_KEY", valueFrom: {secretKeyRef: {key: "key", name: "tls"}}}]' | jy -o $FE
```

Comment:
why did you change this? it is now wrong because the FE variable at the end is not referenced (I defined the variable to shorten the command).